.github/actions/spelling/patterns.txt

```diff
@@ -139,18 +139,8 @@ regexp?\.MustCompile\(`[^`]*`\)
 # ignore long runs of a single character:
 \b([A-Za-z])\g{-1}{3,}\b
 
-# Don't check names in dependabot.yaml reviewers section
+# Don't check names in dependabot.yml reviewers section
 ^\s+reviewers:\s*\[\s*"(?:Nino-K|ericpromislow)"\s*\]
 
-# Directives to avoid extension-related publisher names and repositories:
-
-# Directives to skip a full line
+# Directives to skip the current full line (intended for extension names and their related account names):
```

Comment:
I'm a little confused about why this changes is in this PR. I don't think we need to update our spelling patterns here, but I may have missed something..

Reply:
Because the extension descriptions contain words that are not recognized. See e.g. in https://github.com/rancher-sandbox/rancher-desktop/actions/runs/5942446939

Reply:
Understood. I think my assumption was that the spellchecker would have failed the demo files before this PR was raised, but it must be because the spellchecker only runs against the changes in any given PR and never against the entire repo.

Reply:
I pushed changes working with the spellchecker into the PR at the same time as I had a separate PR that got merged. The one in `main` dealt with recognizing publisher names, etc. The changes in your PR are simplified, whitelisting the two `demo_` JSONish files.